lib/translator/BUILD

```diff
@@ -1,3 +1,17 @@
+cc_library(
+  name = "read_capacity_10_lib",
```

Comment:
Better to have  "//lib/translator:common" as a dependency than to include common.h and .cc as part of the headers and source